CHANGES.md

```diff
@@ -19,6 +19,8 @@
 - Include the content of the line that features the `part-end` MDX directive in
   the output, before that line would've been dropped (#374, #387,
   @Leonidas-from-XIV)
+- Fix parsing of multiline toplevel phrases in .mli files (#394, #397,
+  @Leonidas-from-XIV)
```

Comment:
They sort of are, but this PR does a lot of "under the table reshuffling" so the description in the changelog is essentially just what made me start this endevour. It sort of also preserves the indentation, like #395, but refactors the way this is done.

Another way to describe this could be:

```
- Preserve indentation in multiline OCaml blocks in .mli files and fix multiline cram blocks (#395, #397, @panglesd, @Leonidas-from-XIV)
```

But I feel like one of them is a change and one is pretty much a bugfix, so joining them feels a bit unfortunate.

The problem after working with a PR for too long of a time one looses the user perspective and sees the technical change, but the Changelog is specifically written for users. If you have a suggestion, I'll gladly incorporate it!

Reply:
اريد الدخول الى حسابي الخاص فيسبوك